app/models/user.rb

```diff
@@ -198,6 +198,11 @@ def in_person_pending_profile
     pending_profile if pending_profile&.in_person_verification_pending?
   end
 
+  def has_in_person_enrollment?
+    in_person_enrollments.where(status: [:establishing, :pending]).
+      order(created_at: :desc).first&.selected_location_details.present?
```

Comment:
```suggestion
    pending_in_person_enrollment.present? || establishing_in_person_enrollment.present?
```
We don't really need to check for `selected_location_details` because (for now) [the enrollment cannot be created without them](https://github.com/18F/identity-idp/blob/main/app/controllers/idv/in_person/usps_locations_controller.rb#L43-L46). Also, we plan to make it okay to have an establishing enrollment without location details, so we don't want to tie them together here.

So I think it's more efficient to use the User model's [built in scopes](https://github.com/18F/identity-idp/blob/main/app/models/user.rb#L53-L61) to check for the presence of these enrollments.